.github/workflows/docker.yml

```diff
@@ -136,9 +136,10 @@ jobs:
         if: '!matrix.qemu'
         continue-on-error: ${{fromJson(needs.prepare.outputs.push)}}
         run: |
-          docker run --platform=${{matrix.platform}} --rm \
+          ulimit -s
+          docker run --ulimit stack=$(ulimit -s)000 --platform=${{matrix.platform}} --rm \
```

Comment:
I needed a positively ridiculous stack size to get the tests to pass in my local Docker.

```suggestion
          docker run --ulimit stack=512000000 --platform=${{matrix.platform}} --rm \
```

Reply:
I'm very curious how PHP fibers work, under the hood, beyond my rudimentary knowledge. It'd be a really interesting blog post. 🤔 

Maybe there are some changes to either Frankenphp or PHP itself that can make this better. A 5mb stack is pretty huge...

Reply:
Looking at this with fresh morning eyes. I'm going to write that blog post. I really want to understand what is happening in the stack here.

Reply:
Even with the value you suggested (last commit), it doesn't work :/

Reply:
Nothing changed except the time of day and its failing locally with the same numbers I gave you. This makes no sense.

Reply:
wtaf. I ran it a few more times and it passes
<img width="761" alt="image" src="https://github.com/dunglas/frankenphp/assets/1883296/6d72e37b-4ea9-4295-84bb-9b4e60dc0981">

you can't make this up.

Reply:
Why would it fail the first few times and then pass afterwards.

Reply:
Yes, it's flaky. Even on GHA sometimes test pass, sometimes no.

Could you try to compile PHP with `--disable-fiber-asm` to see if this improves the situation?